de.dlr.sc.virsat.uiengine.ui/src/de/dlr/sc/virsat/uiengine/ui/editor/snippets/general/UiSnippetRoleManagement.java

```diff
@@ -139,10 +192,37 @@ public void widgetDefaultSelected(SelectionEvent e) {
 			}
 		});
 	}
-	
+		
 	@Override
 	protected Set<IMarkerHelper> getMarkerHelpers() {
 		return Collections.singleton(new VirSatProblemMarkerHelper());
 	}
 
+	@Override
+	public void onFeaturesChanged(String[] updatedFeatures) {
+	    EditingDomain editingDomain = AdapterFactoryEditingDomain.getEditingDomainFor(model);
+	    if (editingDomain != null) {
+	        editingDomain.getCommandStack().execute((Command) new RecordingCommand((TransactionalEditingDomain) editingDomain) {

```

Comment:
Maybe just use a SetCommand? Then you don't need to implement you own command here... Or do you have any new functionality here? 
it should probably be something like:
SetCommand.create(editingDomain, selectedDiscipline, RolesPackage.Literals.DISCIPLINE__USERS, updatedFeatures)